src/zcl_abapgit_file_status.clas.abap

```diff
@@ -228,12 +241,12 @@ CLASS ZCL_ABAPGIT_FILE_STATUS IMPLEMENTATION.
 
     " Complete item index for unmarked remote files
     LOOP AT lt_remote ASSIGNING <ls_remote> WHERE sha1 IS NOT INITIAL.
-      identify_object( EXPORTING iv_filename = <ls_remote>-filename
-                                 iv_path     = <ls_remote>-path
-                                 io_dot      = io_dot
-                                 iv_devclass = iv_devclass
-                       IMPORTING es_item     = ls_item
-                                 ev_is_xml   = lv_is_xml ).
+      identify_object( EXPORTING iv_filename     = <ls_remote>-filename
+                                 iv_path         = <ls_remote>-path
+                                 io_dot          = io_dot
+                                 iv_devclass     = iv_devclass
+                       IMPORTING es_item         = ls_item
+                                 ev_is_xml       = lv_is_xml ).
```

Comment:
also no changes here? suggest reverting the call of IDENTIFY_OBJECT to the original code